src/tiling/tiler.rs

```diff
@@ -118,8 +118,7 @@ impl TilingInfo {
     k
   }
 
-  #[inline(always)]
-  pub fn tile_count(&self) -> usize {
+  pub const fn tile_count(&self) -> usize {
```

Comment:
cols and rows aren't constant, the compiler shouldn't allow this.

Reply:
It does though.

Reply:
I think it just means you can construct a const TilingInfo, and then call this method on it and the result is also const.

Reply:
That makes sense.

Reply:
@lu-zero I added back the inline attribute to this function, let me know if that's fine by you.

Reply:
Sure, thank you :)